packages/browser-runtime-core/src/interpreter/interpreter.spec.ts

```diff
@@ -159,24 +159,24 @@ describe('Interpreter', () => {
       ).to.equal(3);
     });
 
-    it.skip('can redeclare a top level function as function', async() => {
-      expect(
-        await testEvaluate(
-          'function f() { return 1; }',
-          'function f() { return 2; }',
-          'f()'
-        )
-      ).to.equal(2);
-    });
-
-    it.skip('can redeclare a top level function as var', async() => {
-      expect(
-        await testEvaluate(
-          'function sum(a, b) { return a + b; }',
-          'var sum = 1'
-        )
-      ).to.equal(1);
-    });
+    // it('can redeclare a top level function as function', async() => {
+    //   expect(
+    //     await testEvaluate(
+    //       'function f() { return 1; }',
+    //       'function f() { return 2; }',
+    //       'f()'
+    //     )
+    //   ).to.equal(2);
+    // });
+    //
+    // it('can redeclare a top level function as var', async() => {
```

Comment:
this is also kinda common and unexpected

Reply:
Will keep it as a skip then and include it in the ticket